labelbox/schema/data_row_metadata.py

```diff
@@ -147,6 +166,13 @@ def _make_name_index(fields: List[DataRowMetadataSchema]):
                 index[f.name] = f
         return index
 
+    @staticmethod
+    def _make_normalized_name_index(fields: List[DataRowMetadataSchema]):
+        index = {}
+        for f in fields:
+            index[f.name] = f
```

Comment:
Aren't options returned as DataRowMetaDataSchemas? So won't this have options and parents at the same level (and also the nesting too - so duplicated options)

Reply:
Discussed on Slack. This is a non-issue